### PR TITLE
[#762] KYC personal flow e2e test

### PR DESCRIPTION
**Pull request by lizonp**
#762

### CODE REVIEW

File: app/components/shared/MultiFileUpload.tsx [hunk 1]
```diff
@@ -23,9 +23,10 @@ interface IProps {
   files: IKycFileInfo[];
   layout: TKycRequestType;
   onDropFile: (file: File) => void;
+  tid?: string;
 }
 
-export const MultiFileUpload: React.SFC<IProps> = ({ files, layout, ...props }) => {
+export const MultiFileUpload: React.SFC<IProps> = ({ files, layout, tid, ...props }) => {
```
Comment (krzkaczor) [inline app/components/shared/MultiFileUpload.tsx]: do not use `tid`. Use `data-test-id` as everywhere else. There are already examples in the codebase how to treat them nicely

File: app/components/shared/forms/formField/FormFieldDate.tsx [hunk 2]
```diff
@@ -9,6 +9,7 @@ import * as errorStyles from "./FormStyles.module.scss";
 interface IProps {
   label: string;
   name: string;
+  tid?: string;
```
Comment (krzkaczor) [inline app/components/shared/forms/formField/FormFieldDate.tsx]: `data-test-id` not tid

File: app/components/shared/forms/formField/FormSelectCountryField.tsx [hunk 3]
```diff
@@ -255,6 +255,7 @@ const VALUES = {
 
 interface IFieldGroup {
   label?: string;
+  tid?: string;
```
Comment (krzkaczor) [inline app/components/shared/forms/formField/FormSelectCountryField.tsx]: same

File: app/components/shared/forms/formField/FormSelectField.tsx [hunk 4]
```diff
@@ -12,19 +12,6 @@ import * as styles from "./FormStyles.module.scss";
 export const NONE_KEY = "__NONE__";
 export const BOOL_TRUE_KEY = "true";
 export const BOOL_FALSE_KEY = "false";
-export const DISABLED_COUNTRIES = [
```
Comment (krzkaczor) [inline app/components/shared/forms/formField/FormSelectField.tsx]: this was unused right?

File: app/components/shared/forms/formField/FormSelectField.tsx [hunk 5]
```diff
@@ -48,6 +35,7 @@ export const unboolify = <T extends {}>(values: T): T => {
 
 interface IOwnProps {
   extraMessage?: string | React.ReactNode;
+  tid?: string;
```
Comment (krzkaczor) [inline app/components/shared/forms/formField/FormSelectField.tsx]: data-test-id

File: cypress/fixtures/example.json [hunk 6]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (krzkaczor) [inline cypress/fixtures/example.json]: i think this one is not used at all.

File: app/components/kyc/__tests__/KYCFlowIndividualManual.spec.e2e.ts [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+  cy.get(tid("access-light-wallet-password-input")).type(password);
+  cy.get(tid("access-light-wallet-confirm")).click();
+};
+
+describe("KYC Personal flow with manual verification", () => {
+  it("went through KYC flow with personal data", () => {
+    goToIndividualKYCFlow();
+    submitIndividualKYCForm(personData);
+    goToIndividualManualVerification();
+    uploadDocumentAndSubmitForm();
```
Comment (krzkaczor) [inline app/components/kyc/__tests__/KYCFlowIndividualManual.spec.e2e.ts]: this test is missing some kind of final assertion. For now we can live with URL assert

File: app/components/kyc/__tests__/KYCFlowIndividualManual.spec.e2e.ts [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+    .wait(1000)
+    .url()
+    .should("eq", `https://localhost:9090${kycRoutes.individualInstantId}`);
+};
+
+const goToIndividualManualVerification = () => {
+  cy.get(tid("kyc-go-to-manual-verification")).click();
+
+  cy
+    .wait(1000)
```
Comment (krzkaczor) [inline app/components/kyc/__tests__/KYCFlowIndividualManual.spec.e2e.ts]: DO NOT USE WAIT. Cypress commands have implicit retry mechanism

File: app/components/kyc/__tests__/KYCFlowIndividualManual.spec.e2e.ts [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+  cy.get(tid("kyc-personal-start-country")).select(person.country);
+
+  cy.get(tid("kyc-personal-start-is-politically-exposed")).select(person.isPoliticallyExposed);
+  cy.get(tid("kyc-personal-start-is-us-citizen")).select(person.isUsCitizen);
+  cy.get(tid("kyc-personal-start-has-high-income")).select(person.hasHighIncome);
+
+  cy.get(tid("kyc-personal-start-submit-form")).click();
+
+  cy
+    .wait(1000)
```
Comment (krzkaczor) [inline app/components/kyc/__tests__/KYCFlowIndividualManual.spec.e2e.ts]: delete .wait everywhere. it should work without them

File: app/components/shared/forms/formField/FormFieldDate.tsx (hunk 2 above)
Comment (lizonp) [reply app/components/shared/forms/formField/FormFieldDate.tsx]: It's changed

Comment (krzkaczor) [reply app/components/shared/forms/formField/FormFieldDate.tsx]: its not! I want to use everywhere `data-test-id` then we can quickly remove it in production mode.

**Pull request merged**